fiboa_cli/convert_utils.py

```diff
@@ -276,6 +281,10 @@ def get_urls(self):
     def read_data(self, paths, **kwargs):
         gdfs = []
         for path, uri in paths:
+            if "*" in path:
+                lst = glob(path)
+                assert len(lst) == 1, f"Can not match {path} to a single file"
+                path = lst[0]
```

Comment:
A comment of the purpose would be helpful, I think. It seems this is for a specific usecase...

Also, glob is a bit tricky on Windows. Works probably in 99% of the cases, but if there's a better solution, we should aim for that ;-)

Reply:
Will add comment. The challenge is that some converter-sources are comprised of many different zip files, and the shapefile naming scheme in these zip files is not consistent/predictable. This will allow a `*.shp` to just match any shapefile... Unpacking/inspecting the zip files in the converter.py file duplicates logic (but keeps the main-file simpler).

Reply:
I guess we can keep it and try it out until we run into issues ;-)

Reply:
Yes, let me know if it fails on windows...